src/main/java/lotto/domain/Lotto.java

```diff
@@ -4,19 +4,19 @@
 import java.util.stream.Collectors;
 
 public class Lotto {
-    LottoPrice price;
-    List<LottoNumbers> lottoNumbers;
+    private final LottoPrice price;
+    private final List<LottoNumbers> lottoNumbers;
 
     public Lotto(LottoPrice price, List<LottoNumbers> lottoNumbers) {
         this.price = price;
         this.lottoNumbers = lottoNumbers;
     }
 
-    public LottoResult getMatchResult(LottoNumbers winningNumbers) {
+    public LottoResult getMatchResult(LottoNumbers winningNumbers, LottoNumber bonusNumber) {
         LottoResult result = new LottoResult();
 
         for (LottoNumbers lottoNumbers : lottoNumbers) {
-            LottoRank rank = lottoNumbers.getMatchedRank(winningNumbers);
+            LottoRank rank = lottoNumbers.getMatchedRank(winningNumbers, bonusNumber);
```

Comment:
```suggestion
            LottoRank rank = winningLotto.getMatchedRank(lottoNumbers);
```
WinningLotto 객체를 도메인 객체로 생각하고 위와 같이 WinningLotto에 등수를 구하는 책임을 부여하는 것은 어떨까?